kong/db/dao/init.lua

```diff
@@ -1011,7 +1025,7 @@ function DAO:select(pk_or_entity, options)
   end
 
   local err
-  if options == nil or options.expand_shorthands == nil then
+  if options == nil or options.expand_shorthands ~= false then
```

Comment:
Don't use `=~ false` or `== true` or the like for boolean variables.  Use `not` instead, if needed.

Reply:
The current logic guards against an explicit `false`. Using `not` in this case would also guard against `nil`.

Reply:
We'd like to make explicit false check here